src/panels/lovelace/special-rows/hui-button-row.ts

```diff
@@ -25,7 +28,15 @@ export class HuiButtonRow extends LitElement implements LovelaceRow {
       throw new Error("Error in card configuration.");
     }
 
-    this._config = { icon: "hass:remote", ...config };
+    this._config = {
+      icon: "hass:remote",
+      tap_action: {
+        action: "call-service",
+        service: config.service,
```

Comment:
This should not be in `hui-button-row` but in `hui-call-service-row`

Reply:
So you want hui-call-service-row to implement its own setConfig? There will be clashes between the two as the types don't overlap.

Reply:
That doesn't matter does it? `call-service-row` will be an individual element that uses `button-row` to be rendered?